scripts/webpack.base.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   plugins: [
     new webpack.DefinePlugin({
       __SCROLLBAR_VERSION__: JSON.stringify(
-        process.env.SCROLLBAR_VERSION || require('../package.json').version,
+        process.env.SCROLLBAR_VERSION || require('../package.json').version
```

Comment:
This cause an error on node v6.11.0 OSX

Reply:
I'd suggest you use higher version of node.js in this project with version managers like [nvm](https://github.com/creationix/nvm). Changing code style needs a huge amount of extra work.